apps/betterangels-backend/clients/tests/test_permissions.py

```diff
@@ -27,25 +27,22 @@ def setUp(self) -> None:
         "user_label, should_succeed",
         [
             ("org_1_case_manager_1", True),
-            ("client_user_1", False),  # Non CM should not succeed
+            ("non_case_manager_user", False),  # Non CM should not succeed
```

Comment:
created a "non cm user" fixture to test these perms, since there's no longer such a thing as a "client user"